frontend/src/ol/staticLayers/layerTypes/tiles.js

```diff
@@ -9,8 +9,10 @@ const tileLayer = (layerData) => {
     featureId: layerData.featureId,
     type: layerData.layerType,
     source: new XYZ({
-      url: layerData.tileDataUrl,
-      attributions: layerData.tileAttributions,
+      url: layerData.tileDataUrl || layerData.geoDataUrl,
+      attributions:
+        layerData.tileAttributions ||
+        (layerData.metadata && layerData.metadata.geoMetadata && layerData.metadata.geoMetadata.description),
```

Comment:
should be attribution, not description, we should keep description for human-friendly explanation :)